addon/.eslintrc.yml

```diff
@@ -0,0 +1,35 @@
+env:
+  browser: true
+
+extends:
+  - eslint:recommended
+  - airbnb-base
+
+root: true
+
+rules:
+  import/newline-after-import: off
+  import/no-extraneous-dependencies: off
+  import/no-unresolved: off
```

Comment:
These feel a bit hacky since we're not explicitly using this plugin here, it seems to be leaking from the parent .eslintrc.yml file. Feels like an ESLint bug, but I'll have to investigate and boil it down to a smaller test case and file upstream, so we'll just ignore these wrongly inherited rules for right meow.